frontend/packages/console-app/src/components/quick-starts/QuickStartController.tsx

```diff
@@ -28,6 +37,22 @@ const QuickStartController: React.FC<QuickStartControllerProps> = ({ quickStart
     (task, index) => activeQuickStartState[`taskStatus${index}`],
   ) as QuickStartTaskStatus[];
 
+  const [contentRef, SetContentRef] = React.useState<HTMLDivElement>(null);
+  const ref = React.createRef<HTMLDivElement>();
+  const shadows = useScrollShadows(contentRef, 100);
```

Comment:
Why not pass ref.current directly here?
```suggestion
  const shadows = useScrollShadows(ref.current, 100);
```

Reply:
The hook will not update unless ref.current is set as a state. This is where `contentRef` comes in and stores ref.current as state

Reply:
Can you move this hook to `QuickStartPanelContent.tsx` and add a prop `contentRef` that allows you to pass a ref function down to the get to the content. That way `QuickStartPanelContent.tsx` can manage the the shadows.